tests/test_state.py

```diff
@@ -3,37 +3,40 @@
 
 import pytest
 from benchcab.utils.state import State, StateAttributeError
+from tempfile import TemporaryDirectory
```

Comment:
Switched the state tests to in-memory, off-topic but it should make things in CI a bit cleaner.